ansible/roles/util-cfg-service/templates/data-info.properties.j2

```diff
@@ -7,6 +7,8 @@ data-info.kifshare-download-template = {{ kifshare.de_url }}
 data-info.max-paths-in-request       = {{ fs_max_paths_in_request }}
 data-info.metadata.base-url          = http://{{ metadata.host }}:{{ metadata.port }}
 data-info.tree-urls.base-url         = http://{{ tree_urls.host }}:{{ tree_urls.port }}
+data-info.kifshare-download-template = {{ kifshare.de_url }}
```

Comment:
Oh, weird. I didn't even think to check since it's unused. I'll remove the extra.